README.md

```diff
@@ -24,34 +38,27 @@ document.getElementById("languageButton").addEventListener("click", (e) => {
 });
 ```
 
-### Where it's used
-
-- NJ DOL, TDI/FLI, MyLeaveBenefits, [Maternity Timeline Tool (Welcome)](https://nj.gov/labor/myleavebenefits/worker/maternity/timeline-welcome.shtml)
-- NJ DOL, TDI/FLI, MyLeaveBenefits, [Maternity Timeline Tool (Tool)](https://nj.gov/labor/myleavebenefits/worker/maternity/timeline-tool.shtml)
-- NJ DOL, TDI/FLI, MyLeaveBenefits, [What happens after I apply?](https://nj.gov/labor/myleavebenefits/worker/resources/claims-status.shtml)
-- NJ DOL, TDI/FLI, MyLeaveBenefits, [Announcing a new way to log in](https://www.nj.gov/labor/myleavebenefits/worker/resources/login-update.shtml)
-- NJ DOL, UI, [Claim Status](https://uistatus.dol.state.nj.us/)
-- Office of Innovation, [Office website](https://innovation.nj.gov/)
-
```

Comment:
I think this is useful to have somewhere for other teams to be aware of (for example, if advocating adding it to Agency X, the team can quickly see where all it's been added already). Maybe I could add it to Ruthie's doc instead since it's less eng related and so it's easier to update?